app/enterprise/1.5.x/introduction/key-concepts.md

```diff
@@ -20,7 +20,14 @@ API gateway authentication is an important way to control the data that is allow
 Authorization is the system of defining access to certain resources. In Kong Enterprise, Role-Based Access Control (RBAC) is the main authorization mode. To define authorization to an API, it is possible to use the ACL Plugin in conjunction with an authentication plugin.
 
 ## Beta
-A Beta designation in Kong software means the functionality of a feature is of high-quality and can be deployed in a non-production environment. **A Beta feature should not be deployed in a production environment.** Some level of support is available for the feature, although user documentation might not be of the highest quality. A Beta feature is made available to the general public for usability testing and to gain feedback about the feature before releasing it as production-ready.
+A Beta designation in Kong software means the functionality of a feature or release version is of high-quality and can be deployed in a non-production environment. Note the following when using a Beta feature or version:
+
+* **A Beta feature or version should not be deployed in a production environment.** 
+* Beta customers are encouraged to engage Kong Support to report issues encountered in Beta testing.  Support requests should be filed with normal priority, but contractual SLA’s will not be applicable for Beta features.
+* Support is not available for data recovery, rollback, or other tasks when using a Beta feature or version. 
+* User documentation might not be of the highest quality. 
```

Comment:
```suggestion
* User documentation might not be complete or reflect complete functionality. 
```

Reply:
OR User documentation might not be complete or reflect all available functionality.